dinstaller-lib/src/settings.rs

```diff
@@ -0,0 +1,141 @@
+use crate::users::{UsersClient,FirstUser};
+use crate::storage::StorageClient;
+use std::{str::FromStr, error::Error};
+
+#[derive(Debug, Default)]
```

Comment:
hmm, so this by default sets all values to `None`. What is reason for it?

Reply:
The reason is that, initially, I did not plan to load all the information from D-Bus (only the part that was needed). But for now, I will load everything, so I am removing the `Option`.